test/handlers/code/shift-enter-last-line.js

```diff
@@ -0,0 +1,15 @@
+/** @jsx h */
+
+export default input => input.break({ shift: true })
```

Comment:
But doing so would be inconsistent with other similar elements (exercises, etc.), and would make editing multi-paragraph notes considerably more difficult. Also, you can always exit a note by pressing enter two times (pressing once in an empty last paragraph will exit it).

Reply:
[NEW ISSUE]

I was thinking about changing this behavior in all of the block elements
I remember that Patryk had a problem with leaving exercise when it was a last element in the document because he didn't know he has to hit enter two times

Reply:
We could also ensure the document always ends (ignoring glossary) with a paragraph, and remove it when serializing if it's empty.

Reply:
I think it would be weird for users and I actually agree with Patryk that current behavior is not intuitive.
Let's discuss it here: https://github.com/openstax-poland/cnx-designer/issues/91